### PR TITLE
Prepare for 12.2.0

### PR DESCRIPTION
**Pull request by nkoenig**
Signed-off-by: Nate Koenig <nate@openrobotics.org>

# 🎈 Release

Preparation for 12.2.0 release.

Comparison to 12.1.0: https://github.com/ignitionrobotics/sdformat/compare/sdformat12_12.1.0...sdf12

Needed by 
* https://github.com/ignitionrobotics/ign-gazebo/pull/1220
* https://github.com/ignitionrobotics/ign-gazebo/pull/1209
* https://github.com/ignitionrobotics/ign-gazebo/pull/1201

## Checklist
- [X] Asked team if this is a good time for a release
- [x] There are no changes to be ported from the previous major version
- [ ] No PRs targeted at this major version are close to getting in
- [x] Bumped minor for new features, patch for bug fixes
- [x] Updated changelog
- [ ] Updated migration guide (as needed)
- [ ] Link to PR updating dependency versions in appropriate repository in [ignition-release](https://github.com/ignition-release/ign-gazebo6-release/pull/11)

**Note to maintainers**: Remember to use **Squash-Merge**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+
+1. Add function to convert Sensor DOM to sdf::ElementPtr
+    * [Pull request #753](https://github.com/ignitionrobotics/sdformat/pull/753)
+    * [Pull request #757](https://github.com/ignitionrobotics/sdformat/pull/757)
+
+1. Support wide angle camera.
+    * [Pull request #744](https://github.com/ignitionrobotics/sdformat/pull/744)
+
+1. Forward ports
+    * [Pull request #756](https://github.com/ignitionrobotics/sdformat/pull/756)
```
Comment (azeey) [inline Changelog.md:19]: Are we not listing all the PRs from the forward ports anymore?

**Review by azeey — APPROVED** [review]

File: Changelog.md (hunk 1 above)
Comment (scpeters) [reply Changelog.md:19]: there was a bit of discussion in https://github.com/ignition-tooling/release-tools/issues/246 but I'm not sure we reached consensus

**Review by mjcarroll — APPROVED** [review]

**Review by chapulina — APPROVED** [review]

File: Changelog.md (hunk 1 above)
Comment (nkoenig) [reply Changelog.md:19]: I can list out all of the PRs. I'll get that up soon.

Comment (nkoenig) [reply Changelog.md:19]: I just added everything in 50d3ee5.

**Review by scpeters — APPROVED** [review]

**Pull request merged**